kitsune/products/jinja2/products/documents.html

```diff
@@ -27,7 +27,9 @@
     {{ topic_sidebar(topics[:10], subtopics, product, topic, subtopic) }}
   </aside>
   <div class="grid_9">
-    {% if show_fx_download %}
+  {% if product.slug == 'firefox' %}
```

Comment:
why only show in firefox product page? can you explain where its decided to show in the firefox products document only?

Reply:
This is the requirement set by @madasan. Perhaps she can shed some light on the decision?

Reply:
@safwanrahman There's two Firefox download buttons: one in the body of the page, one in the header.

The one in the body should be displayed on all Firefox product pages
The one in the header should be displayed on all the other pages (not on the Firefox product ones we already have a button there).

That sounds easy but getting that logic right in the code might be a bit complicated :)